PROVESFlightControllerReference/Components/ModeManager/ModeManager.cpp

```
@@ -506,4 +520,31 @@ F32 ModeManager ::getCurrentVoltage(bool& valid) {
 
+void ModeManager::commandLossCheck() {
+    // Exit early if we haven't received any commands yet (e.g., on first boot) or if we're not in NORMAL mode
+    if (this->m_lastCommandReceivedTime == Fw::ZERO_TIME || this->m_mode != SystemMode::NORMAL) {
+        return;
+    }
+
+    // Protect against concurrent access to command loss state
+    Os::ScopeLock lock(this->m_commandLossMutex);
+
+    // Get command loss period from parameter
+    Fw::ParamValid paramValid;
```

Clang (14.0.6)

[warning] variable 'paramValid' is not initialized

(cppcoreguidelines-init-variables)